server/__tests__/api.test.js

```diff
@@ -76,6 +76,25 @@ describe('mixed-content', () => {
     expect(res.body.suggestedVisualizations).toContain('Map');
   });
 
+  it(
+    'Check /top-countries-with-more-government-websites-with-mixed-content-' +
```

Comment:
With the prettier new configuration, the concatenation of strings to save space is no longer needed. Just remove `+` and run `yarn prettier .`, and everything should be okay

Same for others

Reply:
That's great, done!